### ISSUE TITLE
请问可以使用bert-as-service来做上下句预测这个任务么？

### ISSUE THREAD

**Issue by ky941122** [hanxiao/bert-as-service#313]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [x] Are you running the latest `bert-as-service`?
* [x ] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [ x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#speech_balloon-faq)?
* [ x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

> Some of this information can be collected via [this script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh).

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version:
- Python version:
- `bert-as-service` version: 
- GPU model and memory:
- CPU model and memory:

---

### Description

> Please replace `YOUR_SERVER_ARGS` and `YOUR_CLIENT_ARGS` accordingly. You can also write your own description for reproducing the issue.

I'm using this command to start the server:

```bash
bert-serving-start YOUR_SERVER_ARGS
```

and calling the server via:
```python
bc = BertClient(YOUR_CLIENT_ARGS)
bc.encode()
```

Then this issue shows up:
您好，请问可以使用bert-as-service作为服务来完成上下句预测任务么？我现在使用自己的数据在上下句预测任务上finetune了bert模型，我想依据这个模型起一个服务用来判断两句话是上下句的概率，请问bert-as-service能构建这个服务么？还是只能做单句encoding？

...

**Comment by hscspring** [comment]
不可以。
都没办法用作语言模型，因为 lm 和 w2v 的 loss 不同：
[can the pre-trained model be used as a language model? · Issue #35 · google-research/bert](https://github.com/google-research/bert/issues/35)

**Comment by dandanmylady** [comment]
可以的。1.传入上下句，中间竖线分割。2. 取出first token（cls token）3.然后用此embedding 和 之前fine-tune时的 cls token 上层的dense layer的权重 matmul 4. 计算sigmoid。这样比较丑。比较好的就是照着目前的Bert as service代码加一个新的接口

**Comment by sc89703312** [comment]
> 可以的。1.传入上下句，中间竖线分割。2. 取出first token（cls token）3.然后用此embedding 和 之前fine-tune时的 cls token 上层的dense layer的权重 matmul 4. 计算sigmoid。这样比较丑。比较好的就是照着目前的Bert as service代码加一个新的接口

要这样，但还不如不用bert-as-service，直接在源代码上加方法
